client/src/worldgen_driver.rs

```diff
@@ -41,37 +42,31 @@ impl WorldgenDriver {
             // there's no point trying to generate chunks.
             return;
         }
-        // An extra dodeca::BOUNDING_SPHERE_RADIUS is needed here because we need to generate
-        // enough chunks to ensure that the all chunks within chunk_generation_distance can be filled
-        // with world generation, which requires all nodes surrounding its vertex to be in the graph.
-        traversal::ensure_nearby(
-            graph,
-            &view,
-            chunk_generation_distance + dodeca::BOUNDING_SPHERE_RADIUS,
-        );
-        for (node_id, _) in traversal::nearby_nodes(
-            graph,
-            &view,
-            chunk_generation_distance + dodeca::BOUNDING_SPHERE_RADIUS,
-        ) {
-            graph.ensure_node_state(node_id);
-        }
+        let local_to_view = view.local.inverse();
```

Comment:
nit: let's try to adopt a convention of `x_from_y` when naming transforms. This makes chains much easier to read and sanity-check because it matches the multiplication order: `x_from_y * y_from_z` is a lot more obviously correct than `y_to_x * z_to_y`.

Reply:
Oh, that's an interesting approach. Maybe in a separate PR? The `dodeca` module has a lot of functions we would also want to change.

Reply:
Sure.